sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -909,6 +909,23 @@ class DataSourceV2SQLSuite
     assert(exception.getMessage.contains("The database name is not valid: a.b"))
   }
 
+  test("ShowViews: using v1 catalog, db name with multipartIdentifier ('a.b') is not allowed.") {
+    val exception = intercept[AnalysisException] {
+      runShowTablesSql("SHOW VIEWS FROM a.b", Seq(), expectV2Catalog = false, isShowView = true)
```

Comment:
can we not call `runShowTablesSql`? We only need to check error and it's easier to just write `sql("SHOW VIEWS FROM a.b")`

Reply:
Sure, updated in 2f8074a48c23f3c922958fa95c56cb5ffd64bc35. For same case of `ShowTables` tests in `DataSourceV2SQLSuite`, I'll update in another PR since there have been some other doc/test changes need to be done based on comments of this PR.